libsync/SyncMsgPacket.cpp

```diff
@@ -100,14 +100,19 @@ void SyncStatusPacketWithAlignedTime::encode()
 {
     m_rlpStream.clear();
     // add aligned time into the fields
-    prep(m_rlpStream, StatusPacket, 4) << number << genesisHash << latestHash << alignedTime;

```

Comment:
not a new type of packet?
modify the packet struct will cause compatibility issue?

Reply:
```
if (g_BCOSConfig.version() >= V2_6_0)
    {
        syncMsgPacketFactory = std::make_shared<SyncMsgPacketWithAlignedTimeFactory>();
        // create NodeTimeMaintenance
        m_nodeTimeMaintenance = std::make_shared<NodeTimeMaintenance>();
        syncMaster->setNodeTimeMaintenance(m_nodeTimeMaintenance);
    }
```

only when `supported_version >= v2.6.0`, enable this kind of packet with `alignedTime`, so there will not have compatibility issue.